client/src/components/ElementActions/tests/PublishAction-test.js

```diff
@@ -1,6 +1,12 @@
 /* eslint-disable import/no-extraneous-dependencies */
 /* global jest, describe, it, expect, window */
 
+jest.mock('isomorphic-fetch', () =>
+  () => Promise.resolve({
+    json: () => ({}),
+  })
+);
+
```

Comment:
Taken from [TreeDropdownField-test.js](https://github.com/silverstripe/silverstripe-admin/blob/856955c2a83be9d80d2257f1300082207827242c/client/src/components/TreeDropdownField/tests/TreeDropdownField-test.js#L3-L7) to avoid `ReferenceError: TextEncoder is not defined` when running jest tests. See [this stackoverflow question](https://stackoverflow.com/questions/68468203/why-am-i-getting-textencoder-is-not-defined-in-jest) for more details about this error and its causes.